model/security/get_distinguished_names/structures.smithy

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+//  The OpenSearch Contributors require contributions made to
+//  this file be licensed under the Apache-2.0 license or a
+//  compatible open source license.
+
+$version: "2"
+namespace OpenSearch
+
+@input
+structure GetDistinguishedNames_Input {}
+
+@output
+structure GetDistinguishedNames_Output {
+    distinguishednameslist: DistinguishedNamesList
```

Comment:
This returns a map:
```suggestion
    @httpPayload
    content: DistinguishedNamesMap
```

It is expected this will cause build errors until #127 is merged